index.html

```diff
@@ -12,7 +12,7 @@
     <link rel="stylesheet" href="https://cdn.digital.arizona.edu/lib/arizona-bootstrap/main/css/arizona-bootstrap.min.css">
     <!-- Material Icons CSS -->
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Sharp" rel="stylesheet">
-    <title>AIR Technical Research Applications Development | The bridge between Computer Science and Interdisciplinary research</title>
+    <title>AIR Technical Research Applications Development | The bridge between Computer Science and Interdisciplinary Research</title>
```

Comment:
```suggestion
<title>AIR Technical Research Applications Development | The Bridge Between Computer Science and Interdisciplinary Research</title>
```
For consistency I'd say we should capitalize these as well.